config/Zed/cronjobs/jenkins.php

```diff
@@ -162,10 +159,10 @@
 ];
 
 /* Message broker */
-if (Config::get(MessageBrokerAwsConstants::SQS_RECEIVER_CONFIG)) {
+if (\Spryker\Shared\Config\Config::get(\Spryker\Shared\MessageBroker\MessageBrokerConstants::IS_ENABLED)) {
```

Comment:
Why you did not use namespace here not in `use`?

Reply:
But agree with this point. Changed.